src/code/utils/get-query-param.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (param) ->
+  param = param.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]")
```

Comment:
Well, I could do, but then I'd actually have to spend more time understanding the regex, instead of grabbing this uncommented code that someone committed to CODAP... https://github.com/concord-consortium/codap/blob/0b302ae3927ac2988dfc2cdcd8261202bec58716/apps/dg/core.js#L128-139

😜 😜